### ISSUE TITLE
Review stylesheet inline with a11y guidelines

### ISSUE THREAD

**Issue by danieltj27** [danieltj27/Dark-Mode#90]
The CSS part of the plugin should be reviewed to ensure it meets **_at the very least AA standards_** for accessibility for inclusion within Core. 

Use this tool to check [colour contrast ratios](http://contrast-ratio.com/).

**Comment by danieltj27** [comment]
By using http://contrast-ratio.com/, everything seems to at the very least pass AA standard which is good. The background colour for the body of the page and the grey text passes AAA standard which overall is perfect. So with a little more testing just to make sure, this should be all okay.
